packages/server/graphql/queries/helpers/makeUpcomingInvoice.ts

```diff
@@ -35,7 +35,7 @@ export default async function makeUpcomingInvoice(
     : undefined
 
   const subscription = stripeInvoice.lines.data.find(
-    ({plan}) => plan?.id === StripeManager.PARABOL_TEAM_600
+    ({plan}) => plan?.id === StripeManager.PARABOL_TEAM_800
```

Comment:
+1 I personally would love if we could just call this subscription `PARABOL_TEAM` and not put the amount in the constant name.